spell check fails locally

### Environment

- **Qiskit Nature version**: dcdd1ddc6da85cbecaeae9b3689af1548fd1e8aa
- **Python version**: 3.6, 3.9
- **Operating system**: RHEL 8.5

### What is happening?

The spell check fails locally. This is giving me problems with debugging the CI failure at #462 .

### How can we reproduce the issue?

1. Create a fresh virtual environment
2. Install qiskit-nature from source, along with the dev tools.
3. Run `make spell`.

Result:
Many errors are thrown. Excerpt:
```
% make spell                       
python -m pylint -rn --disable=all --enable=spelling --spelling-dict=en_US --spelling-private-dict-file=.pylintdict --ignore=gauopen qiskit_nature test tools
************* Module qiskit_nature.algorithms.excited_states_solvers.excited_states_eigensolver
qiskit_nature/algorithms/excited_states_solvers/excited_states_eigensolver.py:40:0: C0402: Wrong spelling of a word 'commutativity' in a docstring:
                             commutativity information returned by this method.
                             ^^^^^^^^^^^^^
Did you mean: ''commutative' or 'commutation' or 'commutativities''? (wrong-spelling-in-docstring)
************* Module qiskit_nature.circuit.library.ansatzes.chc
qiskit_nature/circuit/library/ansatzes/chc.py:42:0: C0402: Wrong spelling of a word 'tuples' in a docstring:
            excitations: The list of excitations encoded as tuples of tuples. Each tuple in the list
                                                            ^^^^^^
Did you mean: ''letups' or 'pes' or 'bopes' or 'utils''? (wrong-spelling-in-docstring)
qiskit_nature/circuit/library/ansatzes/chc.py:42:0: C0402: Wrong spelling of a word 'tuples' in a docstring:
            excitations: The list of excitations encoded as tuples of tuples. Each tuple in the list
                                                                      ^^^^^^
Did you mean: ''letups' or 'pes' or 'bopes' or 'utils''? (wrong-spelling-in-docstring)
qiskit_nature/circuit/library/ansatzes/chc.py:42:0: C0402: Wrong spelling of a word 'tuple' in a docstring:
            excitations: The list of excitations encoded as tuples of tuples. Each tuple in the list
                                                                                   ^^^^^
...
```

### What should happen?

No errors are displayed.

### Any suggestions?

_No response_

--- a/.pylintdict
+++ b/.pylintdict
@@ -298,6 +298,7 @@ ollitrault
 onee
 oneeints
 onwards
+onsite
 opflow
 oppenheimer
 optimizers

--- a/.pylintdict
+++ b/.pylintdict
@@ -5,6 +5,7 @@ al
 alanine
 allclose
 alzheimer
+amino
 aminoacid
 aminoacids
 ancilla
@@ -68,6 +69,7 @@ coeff
 coeffs
 colormap
 commutativities
+commutativity
 conda
 conf
 config
@@ -139,6 +141,7 @@ fermion
 fermionic
 fermionicop
 fermions
+filename
 fmt
 fock
 formatter
@@ -147,6 +150,7 @@ fredkin
 gauopen
 gaussian
 gaussiand
+gcc
 getitem
 getter
 gfortran
@@ -165,6 +169,7 @@ hcore
 heidelberg
 heisenberg
 hermite
+hermitian
 hijkls
 hijs
 hilbert
@@ -246,8 +251,9 @@ mcrz
 mct
 mcu
 mcx
-methionine
 metadata
+meth
+methionine
 microsoft
 minao
 mixin
@@ -259,6 +265,7 @@ mol
 morse
 mpl
 mul
+multi
 multinomial
 mypy
 namelist
@@ -301,11 +308,13 @@ otimes
 outpath
 pac
 param
+parameterized
 params
 pauli
 paulis
 paulische
 pca
+peptide
 perturbative
 pes
 phenylalanine
@@ -336,18 +345,22 @@ qiskit's
 qmolecule
 qn
 qregs
+quantization
 quantized
 quartic
 qubit
 qubits
 qutrit
+radians
 rcccx
 rccx
 readme
 realise
 rebranding
 refactor
+refactored
 regs
+reinstall
 repr
 retworkx
 rgb
@@ -409,11 +422,13 @@ sys
 sysctl
 tavernelli
 taylor
+tb
 tdg
 tensored
 terra
 th
 threonine
+timestamp
 toctree
 toffoli
 tol
@@ -423,6 +438,8 @@ translational
 transpilation
 trotterization
 tryptophan
+tuple
+tuples
 twoe
 twoeints
 tyrosine
@@ -434,6 +451,7 @@ uccsd
 ucrx
 ucry
 ucrz
+uhf
 ulimit
 un
 unary
@@ -442,6 +460,7 @@ undirected
 unitaries
 unitstype
 untransforms
+username
 utils
 uvcc
 über
@@ -449,6 +468,7 @@ valine
 vals
 variational
 verbatim
+versa
 vibrational
 vibrationalop
 vibro

